group/edwards25519/point_test.go

```diff
@@ -251,10 +252,31 @@ func TestExpandMessageXOFSHAKE128ShortDST(t *testing.T) {
 	}
 }
 
+func TestPoint_i2OSP(t *testing.T) {
```

Comment:
1. is it possible to separate the 1 byte test with the 2 bytes test ?
2. What is the max number of bytes that can be tested ? Can we create a test for it ?
3. Maybe put the number of bytes as a parameter of the test function.